content/channels/index.textile

```diff
@@ -447,6 +447,72 @@ To publish a single message to multiple channels, make multiple "@publish()@":/a
 <p>A client can choose whether or not to receive messages that they publish themselves using the "@echoMessages@":/api/realtime-sdk/types#client-options property. This is set in @ClientOptions@ with a default value of @true@. Setting @echoMessages@ to @false@ will stop clients receiving the messages they publish themselves. This property is only available using the realtime interface, as it isn't possible to subscribe to messages using the REST interface.</p>
 </aside>
 
+h3(#server-side-batching). Server-side batching [beta]
+
+Server-side batching optimizes message processing and delivery by grouping multiple messages into batches before sending them to subscribers. This feature reduces the overall message count, cutting costs and mitigating the risk of hitting rate limits during high-throughput scenarios.
+
+h4. Key benefits
+
+* Cost efficiency.
+* Reduced system load.
```

Comment:
What do you mean by reduced system load here?

Reply:
I was referring to "Improved Efficiency," but this isn't necessarily the overarching purpose of server-side batching. It's to prevent customers from hitting their rate limit. I've removed "Reduced system load now".